### ISSUE TITLE
Remove stray pair of UL tags around sphere position sliders

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#82]
Just noticed a stray set of UL tags which is likely a left over from when the 2 sliders were actually housed within a list structure.

Please remove the `<ul>` and the `</ul>`.

<img width="706" alt="stray-ul-sphere-positions" src="https://user-images.githubusercontent.com/1902116/51033233-fae97580-15a2-11e9-998e-d8d93cbe0678.png">

**Comment by terracoda** [comment]
@zepumph, likely directly unrelated, but I thought it was worth mentioning here that there is a code clean-up issue in REGULAR as well (see issue https://github.com/phetsims/gravity-force-lab/issues/132#issuecomment-454675666), but it that issue the mass sliders are missing the parent `ul`.

**Comment by zepumph** [comment]
@terracoda, I couldn't find the commit, but this has already been taken care of.
